.github/workflows/build_packages.yaml

```diff
@@ -77,19 +83,20 @@ jobs:
     steps:
     - uses: actions/download-artifact@v2
       with:
-        name: source
+        name: source-23.2.7.2-emqx-2
         path: .
     - name: unzip source code
-      run: Expand-Archive -Path source.zip -DestinationPath ./
+      run: Expand-Archive -Path source-23.2.7.2-emqx-2.zip -DestinationPath ./
     - uses: ilammy/msvc-dev-cmd@v1
-    - uses: gleam-lang/setup-erlang@v1.1.0
+    - uses: gleam-lang/setup-erlang@v1.1.2
       id: install_erlang
       with:
-        otp-version: 24.0.5
+        otp-version: 23.2
```

Comment:
`gleam-lang/setup-erlang` does not yet support the installation of otp24 on windows

Reply:
Please add this as code comment

Reply:
Resolved